packages/instrumenter/src/mutators/mutate.ts

```diff
@@ -24,6 +25,7 @@ export const allMutators: NodeMutator[] = [
   conditionalExpressionMutator,
   equalityOperatorMutator,
   logicalOperatorMutator,
+  methodExpressionMutator,
```

Comment:
Moving https://github.com/stryker-mutator/stryker-js/pull/3508#issuecomment-1136856947 to a thread...

Err:

```
$ npm run test:update

> e2e-tests@0.0.0 test:update
> cross-env CHAI_JEST_SNAPSHOT_UPDATE_ALL=true npm test


> e2e-tests@0.0.0 test
> node tasks/run-e2e-tests.js

node:internal/process/esm_loader:94
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/home/josh/repos/stryker-js/node_modules/rxjs/operators' is not supported resolving ES modules imported from /home/josh/repos/stryker-js/e2e/tasks/run-e2e-tests.js
Did you mean to import rxjs/operators/index.js?
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:414:17)
    at moduleResolve (node:internal/modules/esm/resolve:981:10)
    at defaultResolve (node:internal/modules/esm/resolve:1078:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_UNSUPPORTED_DIR_IMPORT',
  url: 'file:///home/josh/repos/stryker-js/node_modules/rxjs/operators'
}
```

Reply:
Weird! Did you `npm install` inside `e2e` first?

Reply:
Oh! 🤦 that did the trick. And now I see the `npm run e2e` note in `CONTRIBUTING.md`. Great! Running now...